sml/src/sml_file.c

```diff
@@ -27,11 +27,14 @@
 // EDL meter must provide at least 250 bytes as a receive buffer
 #define SML_FILE_BUFFER_LENGTH 512
 
-sml_file *sml_file_parse(unsigned char *buffer, size_t buffer_len) {
+sml_file *sml_file_parse(unsigned char *buffer, 
+						 size_t buffer_len,
+						 const sml_workarounds workarounds) {
```

Comment:
i'm s bit confused that our "constructor" is named `_parse`, but yes, this is the correct place for that code :)